src/transformers/models/maskformer/feature_extraction_maskformer.py

```diff
@@ -328,12 +328,24 @@ def __call__(
         padded up to the largest image in a batch, and a pixel mask is created that indicates which pixels are
         real/which are padding.
 
-        MaskFormer addresses semantic segmentation with a mask classification paradigm, thus input segmentation maps
-        will be converted to lists of binary masks and their respective labels. Let's see an example, assuming
-        `segmentation_maps = [[2,6,7,9]]`, the output will contain `mask_labels =
+        Segmentation maps can be instance, semantic or panoptic segmentation maps. In case of instance and panoptic
+        segmentation, one needs to provide `instance_id_to_semantic_id`, which is a mapping from instance/segment ids
+        to semantic category ids.
+
+        MaskFormer addresses all 3 forms of segmentation (instance, semantic and panoptic) in the same way, namely by
+        converting the segmentation maps to a set of binary masks with corresponding classes.
+
+        In case of instance segmentation, the segmentation maps contain the instance ids, and
+        `instance_id_to_semantic_id` maps instance IDs to their corresponding semantic category.
+
+        In case of semantic segmentation, the segmentation maps contain the semantic category ids. Let's see an
+        example, assuming `segmentation_maps = [[2,6,7,9]]`, the output will contain `mask_labels =
         [[1,0,0,0],[0,1,0,0],[0,0,1,0],[0,0,0,1]]` (four binary masks) and `class_labels = [2,6,7,9]`, the labels for
         each mask.
 
+        In case of panoptic segmentation, the segmentation maps contain the segment ids, and
```

Comment:
Yes that's why they aren't instance IDs ;) classes like "sky" and "grass" aren't instances, which is why I call them segments

Reply:
Yes but it'd be better to clarify in the docstrings, otherwise it comes across as a bit confusing. Especially since both instance and panoptic postprocessing methods return segments_info.

Reply:
Ok I'll clarify that!

Reply:
Should we maybe add those terms (instance IDs and segment IDs) to the glossary and link to it?